content/intro-to-storybook/ember/en/simple-component.md

```diff
@@ -149,51 +142,47 @@ Now we have Storybook setup, styles imported, and test cases built out, we can q
 
 The component is still basic at the moment. First write the code that achieves the design without going into too much detail:
 
-```handlebars
-{{!-- app/components/task.hbs --}}
-
-<div class="list-item {{@task.state}}" data-test-task>
-  <label class="checkbox">
-    <input
-      type="checkbox"
-      disabled
-      name="checked"
-      checked={{this.isArchived}}
-    />
-    <span
-      class="checkbox-custom"
-      data-test-task-archive
-      {{on "click" this.archive}}
-    ></span>
-  </label>
-  <div class="title">
-    <input
-      type="text"
-      readonly
-      value={{@task.title}}
-      placeholder="Input title"
-    />
-  </div>
-  <div class="actions">
-    {{#unless this.isArchived}}
-      <span data-test-task-pin {{on "click" this.pin}}>
-        <span class="icon-star"></span>
-      </span>
-    {{/unless}}
-  </div>
-</div>
+```diff:title=app/components/task.hbs
```

Comment:
@jonniebigodes for new files like this, where the entire things is a diff, it might be better to use the language instead of the diff format.

Reply:
If you could take another look at this and let me know